src/Raven.Server/Documents/Sharding/Subscriptions/OrchestratedSubscriptionConnection.cs

```diff
@@ -76,19 +83,25 @@ protected override void GatherIncludesForDocument(OrchestratorIncludesCommandImp
         protected override async Task<bool> WaitForChangedDocsAsync(AbstractSubscriptionConnectionsState state, Task pendingReply)
         {
             // nothing was sent to the client, but we need to let the shard know he can continue
-            await NotifyShardAboutBatchCompletion();
+            await NotifyShardAboutBatchCompletion(null);
```

Comment:
in cases like that (passing null or bool), it is better to use named argument, so it is easier to understand what are you passing there actually